module/x/peggy/handler.go

```diff
@@ -30,12 +31,37 @@ func NewHandler(keeper Keeper) sdk.Handler {
 			return handleMsgBatchInChain(ctx, keeper, msg)
 		case MsgEthDeposit:
 			return handleMsgEthDeposit(ctx, keeper, msg)
+		case MsgCreateEthereumClaims:
+			return handleCreateEthereumClaims(ctx, keeper, msg)
 		default:
 			return nil, sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, fmt.Sprintf("Unrecognized Peggy Msg type: %v", msg.Type()))
 		}
 	}
 }
 
+func handleCreateEthereumClaims(ctx sdk.Context, keeper Keeper, msg MsgCreateEthereumClaims) (*sdk.Result, error) {
+	// TODO:
+	// auth EthereumChainID whitelisted
+	// auth bridge contract address whitelisted
```

Comment:
isn't adding the bridge contract address itself just a claim? Because the orchestrators are claiming to have observed a peggy contract with the appropriate bytecode and starting validator set.

Reply:
same for the chainID

Reply:
Good question! This is a 🐔 🥚  problem if we rely on the orchestrators. How would they know the the contract address/ chainID and can trust them? I thought about this data as cosmos `Parameters` values which are set in the genesis file and can be updated via gov proposals. For milestone 1, I would expect them to be hard coded if possible....